pkg/manager-server/server_local.go

```diff
@@ -177,52 +162,63 @@ func (c *LocalServerController) discoverUsingSuppliedDevices(s *Storage, devices
 	return nil
 }
 
-func (c *LocalServerController) discoverViaNamespaces(s *Storage) error {
-	paths, err := c.namespaces()
+// Discover NVMe Namespaces
+func (c *LocalServerController) discoverViaNamespaces() error {
+
+	data, err := c.run("nvme list --output-format=json")
```

Comment:
The command shows only the namespaces attached to the Rabbit. Namespaces created as part of a storage pool yet not part of a storage group for the Rabbit would be missed by this command. Is that OK here?

Reply:
Yes, this is scanning for Rabbit namespaces.